ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -55,11 +56,20 @@
     {:collection sanitized-collection
      :warnings warnings}))
 
+(defn- fix-ingest-concept-format 
+   "Fixes formats"
+  [fmt]
+  (if (mt/umm-json? fmt)
```

Comment:
This `if` nest might be more readable as:
```
(if (or 
     (not (mt/umm-json? fmt)) 
     (mt/version-of fmt))
 fmt 
 (str fmt ";version=" (config/ingest-accept-umm-version)))
```
Double-check that I'm not changing your logic, but I'm fairly certain this is the same